packages/expo-blob/android/src/main/java/expo/modules/blob/Blob.kt

```diff
@@ -0,0 +1,186 @@
+package expo.modules.blob
+
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+import expo.modules.kotlin.sharedobjects.SharedObject
+import expo.modules.kotlin.typedarray.TypedArray
+import expo.modules.kotlin.types.EitherOfThree
+import expo.modules.kotlin.types.Enumerable
+import java.io.ByteArrayOutputStream
+import kotlin.math.max
+import kotlin.math.min
+
+internal const val DEFAULT_TYPE = ""
+
+class Blob(
```

Comment:
```suggestion
internal class Blob(
```

nit: personally i'm leaning toward having explicitly "internal". that could prevent symbols from polluting the whole app namespace.